website/docs/configurations.md

```diff
@@ -228,6 +228,7 @@ Flink jobs using the SQL can be configured through the options in WITH clause. T
 | [clustering.async.enabled](#clusteringasyncenabled)                                              | false                         | Async Clustering, default false<br /> `Config Param: CLUSTERING_ASYNC_ENABLED`                                                                                                                                                                                                                                                                                                                            |
 | [clustering.plan.strategy.small.file.limit](#clusteringplanstrategysmallfilelimit)               | 600                           | Files smaller than the size specified here are candidates for clustering, default 600 MB<br /> `Config Param: CLUSTERING_PLAN_STRATEGY_SMALL_FILE_LIMIT`                                                                                                                                                                                                                                                  |
 | [clustering.plan.strategy.target.file.max.bytes](#clusteringplanstrategytargetfilemaxbytes)      | 1073741824                    | Each group can produce 'N' (CLUSTERING_MAX_GROUP_SIZE/CLUSTERING_TARGET_FILE_SIZE) output file groups, default 1 GB<br /> `Config Param: CLUSTERING_PLAN_STRATEGY_TARGET_FILE_MAX_BYTES`                                                                                                                                                                                                                  |
+| [write.insert.cluster](#writeinsertcluster)                                                      | false                         | Inline Clustering, that means it will do the clustering on each write, so the write throughput is low comparing to the async clustering. Default false<br /> `Config Param: INSERT_CLUSTER`                                                                                                                                                                                                               |
```

Comment:
Is this auto-generated?  Configuration docs are automatically updated by `generate_config.sh` script: https://github.com/apache/hudi/tree/asf-site/hudi-utils.  If you see some configs are missing in the latest docs, you may rerun the script and post the updated `.md` file.

Reply:
No, it's not. Ok, i'll fix it by generate_config.sh

Reply:
This config is not missing (https://hudi.apache.org/docs/configurations#Flink-Options-advanced-configs), so i close this PR.

Reply:
This config is not missing (https://hudi.apache.org/docs/configurations#Flink-Options-advanced-configs), so i close this PR.